### ISSUE TITLE
Termination before reconciliation

### ISSUE THREAD

**Issue by purple49** [BenoitMorel/GeneRax#69]
Hi,

I've been trying to run generax in my university's computer cluster, and have been running into issues with generax terminating before performing reconciliation. For context, I have ~150 gene families across 1216 species and have been running some trials with smaller subsets of ~50 gene families. With the smaller subsets, I've tried both DL and DTL models with `--max-sp-radius 5`. While the DL model mostly reaches the 5th optimisation (DTL model usually doesn't get past the 1st), it always terminates before reconciliation. I thought that the program might be struggling to calculate DL/DTL rates for all gene families and ran both models with `--per-family-rates` but there wasn't any difference. I've also tried an even smaller subset of 20, using `--max-sp-radius 3` as well as running the program individually for each gene family using a bash script but nothing seems to work. I've attached the results folder for a run I did with 30 genes (across 1216 species), with the following code for your reference:

`generax --families compiled_families.txt --species-tree RAX_LGG.raxml.bestTree --rec-model UndatedDTL --prefix DTL_results --strategy SPR --per-family-rates `

Thank you for your assistance!

Best,
Edward

[DTL_results.zip](https://github.com/BenoitMorel/GeneRax/files/13454913/DTL_results.zip)

**Comment by BenoitMorel** [comment]
Hi Edward,
I'm on vacation until the end of next week. But I'll let you know once I
can have a look at this.
Benoit

**Comment by BenoitMorel** [comment]
Hi Edward,
Sorry for the delay, I'm back now :)
I have no clue what went wrong. Could you send me the input families so
that I can reproduce? (ideally the smallest dataset you tried that crashed)
Best,
Benoit

**Comment by purple49** [comment]
Hi Benoit,

I've attached the gene trees, alignments, mapping, and species tree files for the smallest subset that I have tried (20 gene families, 1216 species). I've also included the python script I have used to build my family file since my input filenames aren't  compatible with the provided script. Thank you so much for your assistance!

Best,
Edward

[generax_issue.zip](https://github.com/BenoitMorel/GeneRax/files/13554938/generax_issue.zip)
